templates/pages/about.html

```diff
@@ -8,27 +8,59 @@
+				<li><a href="https://www.facebook.com/sparkica?fref=grp_mmbr_list"><img src="{% static "img/matejaverlic.png" %}">Mateja Verlic</a></li>
+				<li><a href="https://www.facebook.com/goranche?fref=grp_mmbr_list"><img src="{% static "img/goranblazic.png" %}">Goran Blazic</a></li>
+				<li><a href="https://www.facebook.com/jure.cuhalev?fref=grp_mmbr_list"><img src="{% static "img/jurecuhalev.png" %}">Jure Cuhalev</a></li>					
+				<li><a href="https://www.facebook.com/svetka.sapelova?fref=grp_mmbr_list"><img src="{% static "img/cvetkasapelova.png" %}">Svetka</li>
+			</ul>
+		</div> 
+		<div class="col-md-4">
+			<h2 class="about">Members</h2>
+			<ul class="about_list">
+				<li><a href='https://www.facebook.com/anuska.sirola?fref=tl_fr_box&pnref=lhc.friends'><img src="{% static "img/Anuska.png" %}">Anamarija Sustic</a></li>
+				<li><a href='https://www.facebook.com/asokica?fref=grp_mmbr_list'><img src="{% static "img/heidi.png" %}">Heidi Pungartnik</a></li>
+				<li><a href='https://www.facebook.com/petra.vidmar.14?fref=grp_mmbr_list'><img src="{% static "img/petra.png" %}">Petra Vidmar</a></li>
+				<li><a href='https://www.facebook.com/nina.ninchy2?fref=grp_mmbr_list'><img src="{% static "img/ninaninchy.png" %}">Nina Ninchy</a></li>
+					<li><a href='https://www.facebook.com/mojca007?fref=grp_mmbr_list'><img src="{% static "img/mojca.png" %}">Mojca</a></li>
+				<li><a href='https://www.facebook.com/dobai.livia.5?fref=grp_mmbr_list'><img src="{% static "img/liviadobai.png" %}">Livia Dobai</a></li>
+				<li><a href='https://www.facebook.com/masa.c.logar?fref=grp_mmbr_list'><img src="{% static "img/masalogar.png" %}">Masa Cernovsek Logar</a></li>	
+				</ul>
 		</div>
 		<div class="col-md-4">
-		<a href="http://codecatz.github.io/">
-			<p>The "OTHER" homepage</p>
-		</a>
+			<h2 class="transparent">Members</h2>
+			<ul class="about_list">
+				<li><a href="https://www.facebook.com/nina.malovrh.1?fref=grp_mmbr_list"><img src="{% static "img/ninamalovrh.png" %}">Nina Malovrh</a></li>
+				<li><a href="https://www.facebook.com/profile.php?id=1186838569&fref=grp_mmbr_list"><img src="{% static "img/hanaisnouk.png" %}">Hana Isnouk</a></li>
+				<li><a href="https://www.facebook.com/pijab1?fref=grp_mmbr_list"><img src="{% static "img/pijabalaban.png" %}">Pija Balaban</a></li>
+				<li><a href="https://www.facebook.com/nejc.jelovcan?fref=grp_mmbr_list"><img src="{% static "img/nejcjelovcan.png" %}">Nejc Jelovcan</a></li>
+				<li><a href="https://www.facebook.com/natasa.kristan.35?fref=grp_mmbr_list"><img src="{% static "img/natasamori.png" %}">Natasa Mori</a></li>
+				<li><a href="https://www.facebook.com/barbara.skubic.9?fref=grp_mmbr_list"><img src="{% static "img/barbaraskubic.png" %}">Barbara Skubic</a></li>
+				<li><a href="https://www.facebook.com/maria.cankar?fref=grp_mmbr_list"><img src="{% static "img/marijacankar.png" %}">Marija Cankar</a></li>
+			</ul>	
 		</div>
 	</div>
-
+	
```

Comment:
a lone tab on an otherwise empty line... :smile: 
(nothing major, but if you're fixing the p tags, please remove this tab character as well :wink: )